src/config/config.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
     database: DB_NAME_TEST,
     host: DB_HOST,
     dialect: 'postgres',
-    port: DB_PORT
+    port: DB_PORT,
+    logging: false
```

Comment:
You forgot to remove logging: false line. @Mnickii said that logs should be there for the debugging purpose. Please remove it.

Reply:
@habinezadalvan I have removed it thank You for reminding me

Reply:
great